ckanext/harvest/tests/test_action.py

```diff
@@ -307,6 +307,58 @@ def test_update(self):
         assert source.type == source_dict['source_type']
 
 
+class TestHarvestSourceActionPatch(HarvestSourceActionBase):
+
+    @classmethod
+    def setup_class(cls):
+
+        cls.action = 'harvest_source_patch'
+
+        super(TestHarvestSourceActionPatch, cls).setup_class()
+
+        # Create a source to udpate
+        source_dict = cls.default_source_dict
```

Comment:
It's really great to have this improvement. On CKAN tests, we've changed our style a bit over the past couple of years - see http://docs.ckan.org/en/latest/contributing/testing.html#guidelines-for-writing-new-style-tests and are not keen on creating fixtures that are shared between tests like this.

To help with this, you can quickly setup fixtures like a HarvestSource in each test using the new factories. See example here:
https://github.com/ckan/ckanext-harvest/blob/master/ckanext/harvest/tests/test_action.py#L311-L316